### PR TITLE
Docs: broken hyperlink

### PR DESCRIPTION
**Pull request by mjseaman**
This link is broken. Could you help update it?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docusaurus/docs/get-started/get-started.mdx [hunk 1]
```diff
@@ -163,10 +163,12 @@ The file structure should look like this:
 └── tsconfig.json
 ```
 
+THIS LINK IS BROKEN:
 For more information about these files, refer to [Folder structure](get-started/folder-structure/).
 
 ## Build and run your plugin in Docker
 
+THIS LINK IS ALSO BROKEN
 With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](get-started/set-up-development-environment/).
```
Comment (mjseaman) [inline docusaurus/docs/get-started/get-started.mdx:172]:
```suggestion
With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](developer-tools/get-started/set-up-development-environment/).
```

File: docusaurus/docs/get-started/get-started.mdx [hunk 2]
```diff
@@ -163,10 +163,12 @@ The file structure should look like this:
 └── tsconfig.json
 ```
 
+THIS LINK IS BROKEN:
 For more information about these files, refer to [Folder structure](get-started/folder-structure/).
 
 ## Build and run your plugin in Docker
 
+THIS LINK IS ALSO BROKEN
```
Comment (mjseaman) [inline docusaurus/docs/get-started/get-started.mdx:171]:
```suggestion
```

File: docusaurus/docs/get-started/get-started.mdx [hunk 3]
```diff
@@ -163,10 +163,12 @@ The file structure should look like this:
 └── tsconfig.json
 ```
 
+THIS LINK IS BROKEN:
```
Comment (mjseaman) [inline docusaurus/docs/get-started/get-started.mdx:166]:
```suggestion

```

File: docusaurus/docs/get-started/get-started.mdx [hunk 4]
```diff
@@ -163,11 +163,11 @@ The file structure should look like this:
 └── tsconfig.json
 ```
 
-For more information about these files, refer to [Folder structure](get-started/folder-structure/).
+For more information about these files, refer to [Folder structure](developer-tools/get-started/folder-structure/).
```
Comment (jackw) [inline docusaurus/docs/get-started/get-started.mdx:166]:
```suggestion
For more information about these files, refer to [Folder structure](/get-started/folder-structure/).
```

**Comment by leventebalogh** [comment]
Hmm, for me these links seem working on https://grafana.com/developers/plugin-tools/:
- https://grafana.com/developers/plugin-tools/get-started/folder-structure/
- https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment

Or am I missing something?

File: docusaurus/docs/get-started/get-started.mdx [hunk 5]
```diff
@@ -163,11 +163,11 @@ The file structure should look like this:
...
 
 ## Build and run your plugin in Docker
 
-With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](get-started/set-up-development-environment/).
+With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](developer-tools/get-started/set-up-development-environment/).
```
Comment (jackw) [inline docusaurus/docs/get-started/get-started.mdx:170]:
```suggestion
With the `create-plugin` tool, you can use a Docker container to simplify the configuration, loading, and development processes. For more information, refer to [Set up development environment](/get-started/set-up-development-environment/).
```

**Review by jackw — APPROVED** [review]
Thanks so much for spotting these and opening a PR to fix them. I've spotted a couple of others that I'm working on a fix for.

I've left a couple of suggestions which use `/` at the start so the links resolve against the docusaurus `baseUrl` config property.

**Comment by jackw** [comment]
Hey @mjseaman I've noticed a couple in other places. I'm in the middle of 

> Hmm, for me these links seem working on https://grafana.com/developers/plugin-tools/:
> 
> * https://grafana.com/developers/plugin-tools/get-started/folder-structure/
> * https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment
> 
> Or am I missing something?

Those urls exist but the links @mjseaman has spotted resolve to:
- https://grafana.com/developers/get-started/set-up-development-environment/
- https://grafana.com/developers/get-started/folder-structure/

neither of which exist due to missing `plugin-tools` in the url.

**Comment by jackw** [comment]
And now I've force refreshed the entire site and it's working in production. 🤔

**Comment by josmperez** [comment]
This link works now on my local build. Thanks for noticing this issue and fixing it. Merging.

**Pull request merged**